python/setup.py

```diff
@@ -77,14 +77,15 @@
 USER_SCRIPTS_PATH = os.path.join(SPARK_HOME, "sbin")
 DATA_PATH = os.path.join(SPARK_HOME, "data")
 LICENSES_PATH = os.path.join(SPARK_HOME, "licenses")
+DOCKER_PATH = os.path.join(SPARK_HOME, "resource-managers/kubernetes/docker/src/main/dockerfiles/spark")
```

Comment:
Why it's needed?

Reply:
These files are copied into the build as `deps/k8s` and the `entrypoint.sh` is called in the Affirm batch Dockerfile 

https://github.com/Affirm/all-the-things/blob/87ab4991fd07115c6f97fbceb735ab13510a15a9/deployable/batch-infra-tasks/src/Dockerfile#L9-L10